.github/workflows/hono_deploy_production.yml

```diff
@@ -25,26 +25,10 @@ jobs:
         run: npm ci
         working-directory: hono
 
-      - name: Migrate D1
-        id: migrate
-        uses: cloudflare/wrangler-action@v3
-        with:
-          apiToken: ${{ secrets.CF_API_TOKEN }}
-          command: d1 migrations apply prod-shafa-hono-db --env production
-          workingDirectory: hono
-
       - name: Deploy Hono
```

Comment:
This needs to be updated to match the new cf_deploy_hono_stage job in `hono_integration`

Reply:
Resolved.